bnb/getBNBData

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+debug = False
+
+import io, os, re, requests, sys, time, zipfile
+from bs4 import BeautifulSoup
+
+maxretry = 3
+datadir = './BNBDaten' if len(sys.argv) < 2 else sys.argv[1]
+urlbase = 'https://www.bl.uk'
+urlpath = '/collection-metadata/new-bnb-records'
```

Comment:
Shouldn't we write the constants as all-caps?

Reply:
Nope. ;)